docs/resources/mysql.sql

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
lets change this to 2024

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/a00b9b419db2aa1ba4198ff1a8f77e6bd545b1c4